### PR TITLE
Fix attribute documentation for Divider

### PR DESCRIPTION
**Pull request by jnysteen**
The "Type" attribute is incorrectly used - "DividerType" is the correct attribute

### CODE REVIEW

File: docs/_docs/components/divider.md [hunk 1]
```diff
@@ -49,4 +49,4 @@ Default line style is solid.
 | Name        | Type                                                                   | Default   | Description                                        |
 |-------------|------------------------------------------------------------------------|-----------|----------------------------------------------------|
 | DividerType | [DividerType]({{ "/docs/helpers/enums/#dividertype" | relative_url }}) | `Solid`   | Specifies horizontal line style variants.          |
-| Text        | string                                                                 | null      | Label that will appear between the solid lines.    |
\ No newline at end of file
+| Text        | string                                                                 | null      | Label that will appear between the solid lines.    |
```
Comment (jnysteen) [inline docs/_docs/components/divider.md]: This change is Github being Github - probably some line ending being changed. Take it or leave it :)

Comment (stsrki) [reply docs/_docs/components/divider.md]: I will take it :)

**Comment by stsrki** [comment]
LGTM. Thank you!

**Comment by jnysteen** [comment]
> 
> 
> LGTM. Thank you!

Thank you for this very solid library 👍

**Pull request merged**